templates/etc/admin/admin.conf.erb

```diff
@@ -0,0 +1,87 @@
+# The pulp server configuration
```

Comment:
I'm not disagreeing, sometimes I just stop trying to propagate it but I'll just do it all.